.github/scripts/check_labels.py

```diff
@@ -10,20 +11,46 @@
     GitRepo,
 )
 from trymerge import (
+    _fetch_url,
     gh_post_pr_comment,
     GitHubPR,
 )
 
 
+ERR_MSG = ("This PR needs a label. If your changes are user facing and intended to be a "
+           "part of release notes, please use a label starting with `release notes:`. If "
+           "not, please add the `topic: not user facing` label. For more information, see "
+           "https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work.")  # noqa: E501  pylint: disable=line-too-long
```

Comment:
Tweaking the wording & taking advantage of github's markdown format

```suggestion
ERR_MSG = (
           "# This PR needs a label\n"
           "If your changes are user facing, please add a label starting with `release notes:\n\n`",
           "If not, please add the `topic: not user facing` label.\n\n",
           "For more information, see "
           "https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work.")  # noqa: E501  pylint: disable=line-too-long
```